src/test/java/org/llorllale/cactoos/matchers/HasValuesTest.java

```diff
@@ -27,19 +27,17 @@
 
 package org.llorllale.cactoos.matchers;
 
+import java.io.IOException;
 import org.cactoos.list.ListOf;
-import org.hamcrest.Description;
-import org.hamcrest.StringDescription;
-import org.hamcrest.core.IsEqual;
+import org.cactoos.text.Joined;
 import org.junit.Test;
 
 /**
  * Test case for{@link HasValues}.
  *
  * @since 1.0.0
+ * @checkstyle JavadocMethodCheck (500 lines)
```

Comment:
@victornoel I think it's preferable to add Javadoc for the new test (the old ones had it), instead of waiving the requirement.

Reply:
@scristalli see https://github.com/llorllale/cactoos-matchers/pull/134#discussion_r303199530